src/pages/contributors/valentin-delaye.adoc

```diff
@@ -0,0 +1,54 @@
+= Valentin Delaye
+:page-name: Valentin Delaye
+:page-linkedin: 
+:page-twitter: 
+:page-github: jonesbusy
+:page-email: 
+:page-image: avatar/valentin-delaye.png
+:page-pronouns: He/Him/His
+:page-location: Lausanne, Switzerland
+:page-firstcommit: 2009
+:page-datepublished: 2024-02-07
+:page-featured: true
+:page-intro: Valentin is a Jenkins contributor and plugin maintainer who's passionate about technology and programming. This passion is so great that it has spread as both his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is not only about consuming and taking advantage of it, but also contributing to it.
```

Comment:
```suggestion
:page-intro: Valentin is a Jenkins contributor and plugin maintainer who is passionate about technology and programming. This passion is so great that it has spread as both his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is not only about consuming and taking advantage of it, but also contributing to it.
```
Nit, I might be wrong but this abbreviation here sounds a bit strange to me.

Reply:
I definitely get what you mean here @lemeurherve, I'm okay with changing it accordingly. It still makes sense and is the same message, so there's no difference and makes it easier for others.